test/tbb/test_tbbbind.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+    Copyright (c) 2019-2025 Intel Corporation
+    Copyright (c) 2025 UXL Foundation Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+//! \file test_tbbbind.cpp
+//! \brief Test for TBBbind library, covers [configuration.debug_features]
+
+#define TRY_BAD_EXPR_ENABLED 1
+#if _WIN32 || _WIN64
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
+#include "oneapi/tbb/global_control.h"
+#include "oneapi/tbb/info.h"
+#include "common/test.h"
+#include "common/utils_assert.h"
+
+using namespace tbb::detail::r1;
+
+// no need to do it in the test
+#define __TBB_SKIP_DEPENDENCY_SIGNATURE_VERIFICATION 1
+
+#include "../../src/tbb/dynamic_link.cpp"
+
+class binding_handler;
+    // dummy implementation, not used
+class governor {
+    public:
+        static unsigned default_num_threads () { return 0; }
+};
+
+#include "../../src/tbb/load_tbbbind.cpp"
```

Comment:
Why we can't directly inject `dynamic_link.cpp` and `governor.cpp` here instead of creating the separate source file? Or perhaps using `test/common/inject_scheduler.h` header

Reply:
Direct inclusion is probably better, while requiring some manual work.

Reply:
And now I think that creation of isolated component like dynamic_link (with a possibility to just `#include` it) is better, but it's unclear how simple is that.